pkg/limayaml/limayaml.go

```diff
@@ -142,6 +143,12 @@ type HostResolver struct {
 	Hosts   map[string]string `yaml:"hosts,omitempty" json:"hosts,omitempty"`
 }
 
+type CACertificates struct {
+	RemoveDefaults bool     `yaml:"remove_defaults,omitempty" json:"remove_defaults,omitempty"`
+	Files          []string `yaml:"files,omitempty" json:"files,omitempty"`
+	Certs          []string `yaml:"certs,omitempty" json:"certs,omitempty"`
+}
+
```

Comment:
https://github.com/lima-vm/lima/blob/939f79b4bd7e1e391e416bf96b4e325fddbbdc98/pkg/limayaml/defaults.go#L75 needs to be updated too?

Reply:
Yes, I mentioned that in my review at https://github.com/lima-vm/lima/pull/783#pullrequestreview-935895549

Reply:
Should be fixed. For the CA files and certs, I'm appending and de-duping them.